client/analytics/components/report-table/index.js

```diff
@@ -101,19 +109,27 @@ class ReportTable extends Component {
 		const filteredHeaders = this.filterShownHeaders( headers, userPrefColumns );
 
 		return (
-			<TableCard
-				downloadable
-				headers={ filteredHeaders }
-				ids={ ids }
-				isLoading={ isLoading }
-				onQueryChange={ onQueryChange }
-				onColumnsChange={ this.onColumnsChange }
-				rows={ rows }
-				rowsPerPage={ parseInt( query.per_page ) || QUERY_DEFAULTS.pageSize }
-				summary={ summary }
-				totalRows={ totalResults }
-				{ ...tableProps }
-			/>
+			<Fragment>
+				<div
+					className="woocommerce-report-table__scroll-point"
+					ref={ this.scrollPointRef }
```

Comment:
Are there issues with adding this ref to the `TableCard` component directly?

Reply:
The main reason is that the scroll-point needs to be rendered on top of the table, so the browser scrolls some pixels above where the table starts. We would not be able to do that if the ref was assigned directly to the table.
![image](https://user-images.githubusercontent.com/3616980/56498786-92942380-6503-11e9-85f4-88a1ec68f48b.png)